src/App.vue

```diff
@@ -103,6 +107,11 @@
       ></topic-form>
     </div>
   </modal>
+
+  <modal v-model:opened="showSettings" @close="showSettings = false">
```

Comment:
booleans on the modal - just can be used with the v-model no need for the additonal close event (it was only needed where the trigger isn't a boolean)

Reply:
If you don't add the `@onclose` call there than clicking the X button will close the modal but the next click on the open modal button won't show it. So you need to do like a double click in that case to open it up.

Reply:
ohhh ok then I guess the modal component itself doesn't update the synced property gotcha